src/public/location.ts

```diff
@@ -0,0 +1,93 @@
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError, ErrorCode } from './interfaces';
+import { ensureInitialized, sendMessageRequestToParent, isAPISupportedByPlatform } from '../internal/internalAPIs';
+import { FrameContexts } from './constants';
+
+/**
+ * This is the SDK version when location APIs (getLocation and showLocation) are supported.
+ */
```

Comment:
Generally we follow nameSpace approach to club similar APIs together. Can you please move Location related API under Location namespace. please take a look at this https://github.com/OfficeDev/microsoft-teams-library-js/blob/master/src/public/settings.ts

Reply:
@sachingupta, moved the APIs under location namespace.